src/components/modules/DetailedPage/ProductHeader/StyledProductHeader.js

```diff
@@ -17,6 +27,11 @@ export const Row = styled.div`
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+
+  & > ${Tags} {
+    margin-top: 1rem;
+    margin-left: 1rem;
+  }
```

Comment:
Tags component can be used in other contexts other than the product header, the problem I'm trying to fix with this is that Tags in its base styledcomponent has margin top and left that gets applied whatever the context was so I only applied margin top and left when they're children of `Row`